### ISSUE TITLE
Local build agents

### ISSUE THREAD

**Issue by Klaven** [null-channel/AssemblyLine#3]
If supporting a local build agent is an MVP feature...
AND we are using containers (either per step or just one huge build container) ...

What container runtimes do we want to support first?

I'm assuming Docker

**Comment by Klaven** [comment]
@RussellWaite are we just targeting the CRI right now yes for PoC?

**Comment by RussellWaite** [comment]
@Klaven yes for local, not sure what that means for kubernetes hosted build agents as direct CRI would be outside of kubernetes wouldn't it?
